src/Raven.Server/Indexing/VoronBufferedInput.cs

```diff
@@ -22,6 +22,46 @@ public class VoronBufferedInput : BufferedIndexInput
 
     private bool _isOriginal = true;
 
+    /// <summary>Reads a long stored in variable-length format.  Reads between one and
+    /// nine bytes.  Smaller values take fewer bytes.  Negative numbers are not
+    /// supported. 
+    /// </summary>
+    public override long ReadVLong(IState state)
+    {
+        if (bufferPosition + 9 < bufferLength)
```

Comment:
I'm pretty sure that for `vlong`, you have a max of _10_ bytes, not 9.

Reply:
https://github.com/ravendb/lucenenet/blob/60d42a72ed5d20645c1244217dd2b5ea59241420/src/Lucene.Net/Store/IndexInput.cs#L108-L122

I took `9` from the docs.

Reply:
Sorry, you are correct, it is `ulong` that may need up to 10 bytes.